jena-core/src/main/java/org/apache/jena/rdf/model/ResourceFactory.java

```diff
@@ -92,6 +92,15 @@ public static Resource createResource(String uriref) {
         return instance.createResource(uriref);
     }
     
+    /** 
+     * create a new resource representing an RDF* triple term.
```

Comment:
s/create/Create

Reply:
Many of the javadoc comments are like that.  ("Answer" but "create")

Redid the commit : I've gone through the javadoc on all methods and tidied up.

Reply:
Sorry, I had looked only at a couple comments available in the GitHub interface.

Reply:
No problem - usual balance between focused changes for the ticket addressed and a desire to sort out things in passing.